packages/jaeger-ui/src/utils/redux-form-field-adapter.js

```diff
@@ -14,22 +14,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { Popover } from 'antd';
+import cx from 'classnames';
 import * as React from 'react';
 
+import './redux-form-field-adapter.css';
+
 export default function reduxFormFieldAdapter(
   AntInputComponent: Class<React.Component<*, *>>,
   onChangeAdapter: () => void
 ) {
   return function _reduxFormFieldAdapter(props: any) {
-    const { input: { value, onChange }, children, ...rest } = props;
+    // inputRest includes necessary props such as onBlur and value
+    const { input: { onChange, ...inputRest }, children, ...rest } = props;
```

Comment:
If you destruct `className` from `props`, you can add it to the `cx(...)` call. This will allow for the `.SearchForm--validatedInput` class to be added and avoid overriding `.ant-input` styles.

Also, for accessibility to folks who don't work primarily in JS, we've been using `Boolean(value)` instead of `!!value` to cast to a boolean.

Seems like it might be useful to have `isInvalid` only `true` when [`!rest.meta.active`](https://redux-form.com/7.0.3/docs/api/field.md/#-code-meta-active-boolean-code-) ?

Reply:
I'm fine with using `Boolean(value)` instead of `!!value`, but I was basing it off of [airbnb's stye guide regarding 'no new'](https://github.com/airbnb/javascript#coercion--booleans) we should make a note in our style guide if we want to deviate from airbnb.

Reply:
Agreed. [This is what I have](https://github.com/jaegertracing/jaeger-ui/blob/style-guide-wip/CONTRIBUTING.md#code-style) so far for style-guide content. We should merge it with what you're working on.